lib/kay/src/slot_map.rs

```diff
@@ -55,19 +65,23 @@ impl SlotMap {
         }
     }
 
+    /// Set the indices at the ID
     pub fn associate(&mut self, id: usize, new_entry: SlotIndices) {
         let entry = self.entries.at_mut(id);
         entry.clone_from(&new_entry);
     }
 
+    /// Lookup the indices at the ID
     pub fn indices_of(&self, id: usize) -> &SlotIndices {
         self.entries.at(id)
     }
 
+    /// Mark an ID as free for reuse
     pub fn free(&mut self, id: usize, version: usize) {
         self.free_ids_with_versions.push((id, version + 1));
     }
 
+    /// Get an ID which is currently in use ?????
```

Comment:
Get a random ID which is in use (to be able to send a message to a random instance in a Swarm)